src/status_im/chat/styles/input/input.cljs

```diff
@@ -82,8 +82,8 @@
    :ios                 {:line-height min-input-height
                          :left        (+ 10 left)}})
 
-(defnstyle seq-input-text [left]
-  {:min-width           200
+(defnstyle seq-input-text [left container-width]
```

Comment:
That wouldn't help at all, as you need to make `:min-width` the same as container-width minus the place take by `/command` prefix from left. 
200 was just arbitrary number, it's better to compute remaining place left for input.